moveit_core/collision_detector_fcl_description.xml

```diff
@@ -5,4 +5,4 @@
       FCL Collision Detector, default for MoveIt.
     </description>
   </class>
-</library>
```

Comment:
ideally you revert this change, as it unnecessarily dirties the git history of this FCL file.